spec/pheromone/publishable_spec.rb

```diff
@@ -70,83 +70,87 @@ def type
           topic: 'topic5',
           if: ->(data) { data.condition },
           message: :message
-        }
+        },
+        {
+          event_types: [:create],
+          topic: 'topic6',
+          message: :message,
+          metadata: { test: :metadata }
+        },
       ]
     end
   end
 
+  let(:create_message) do
+    {
+      event: 'create',
+      entity: 'PublishableModel',
+      timestamp: '2015-03-12T00:30:00.000Z',
+      blob: { name: 'sample' }
+    }.to_json
+  end
+
+  let(:update_message) do
+    {
+      event: 'update',
+      entity: 'PublishableModel',
+      timestamp: '2015-03-12T00:30:00.000Z',
+      blob: { name: 'new name' }
+    }.to_json
+  end
+
+  let(:metadata_message) do
+    {
+      event: 'create',
+      entity: 'PublishableModel',
+      test: 'metadata',
+      timestamp: '2015-03-12T00:30:00.000Z',
+      blob: { name: 'sample' }
+    }.to_json
+  end
+
   let(:model_create_messages) do
     [
-      {
-        event: 'create',
-        entity: 'PublishableModel',
-        timestamp: '2015-03-12T00:30:00.000Z',
-        blob: { name: 'sample' }
-      }.to_json,
-      {
-        event: 'create',
-        entity: 'PublishableModel',
-        timestamp: '2015-03-12T00:30:00.000Z',
-        blob: { name: 'sample' }
-      }.to_json,
+      create_message,
+      create_message,
       {
```

Comment:
since you're extracting this out, you could do `create_message.merge!(blob: {title: 'title'})` for this one